k8s/nginx/production/docs-vs.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: docs-virtual-server
   namespace: production
 spec:
-  host: docs.airqo.net
+  host: wiki.airqo.net
```

Comment:
[switch docs subdomains to wiki](https://github.com/airqo-platform/AirQo-api/pull/1642/commits/bd663a8b432a452925f4de7151f745b2146c223a)